lightning/src/util/config.rs

```diff
@@ -228,6 +236,7 @@ impl_writeable_tlv_based!(ChannelConfig, {
 	(4, announced_channel, required),
 	(6, commit_upfront_shutdown_pubkey, required),
 	(8, forwarding_fee_base_msat, required),
+	(10, max_outbound_dusted_htlc_msat, required),
```

Comment:
Making this `required` means we cannot read serialized configs from old versions. Can you take https://git.bitcoin.ninja/index.cgi?p=rust-lightning;a=commit;h=fcf9ac5ca6055254955fca9a1402b24168aeab14 and use `(default_value, 25_350_000)` instead of `required`?

Reply:
Does it present another burden for the user, namely to override the default value after deser if user is willingly to enforce its own setting ?

Reply:
Yes, we'll need them to update the field after loading. Sadly we currently don't have any mechanism to update channel config after load at all, but it's something we'll need to as soon to allow updating channel channel relay fees soon.